docs/UserGuide.md

```diff
@@ -180,16 +180,47 @@ edit 1 n/John te/john_123 t/TA
 
 Edits the Profile linked to the Address Book.
 
+
 Format:
 
 ```
 edit profile [n/<NAME>] [te/<TELEGRAM>] [g/<GITHUB>]
 ```
 
-Example:
-```
-edit profile te/bob_osum
-```
+:bulb: TIP:
+<br/>
+
+1. Name Conventions:  
+   • Should only contain alphabetical characters.
+   <br/>
+2. GitHub Username Conventions:
+   <br/>
+   • May only contain alphanumeric characters or hyphens.
+   <br/>
+   • Cannot have multiple consecutive hyphens.
+   <br/>
+   • Cannot begin or end with a hyphen.
+   <br/>
+   • Must be between 4 to 39 characters long
+   <br/>
+3. Telegram Handle Conventions:
+   <br/>
+   • Can contain the letters a-z, digits 0-9 and underscores.
+   <br/>
+   • Must be 5 to 39 characters long.
```

Comment:
This should be 5 to 32 characters long!
![image](https://user-images.githubusercontent.com/72136453/140630499-4875cef4-479d-43f8-9523-9b8dcf97c371.png)

Reply:
I mentioned 39 because an earlier part of the UG which stated the telegram handle conventions said 5 to 39 characters long. Will update both parts!